src/wp-includes/block-bindings/sources/pattern.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * The "pattern" source for the Block Bindings API. This source is used by the
+ * Pattern Overrides.
+ *
+ * @since 6.5.0
+ * @package WordPress
+ */
+function pattern_source_callback( $source_attrs, $block_instance, $attribute_name ) {
+	if ( ! _wp_array_get( $block_instance->attributes, array( 'metadata', 'id' ), false ) ) {
+		return null;
+	}
+	$block_id = $block_instance->attributes['metadata']['id'];
+	return _wp_array_get( $block_instance->context, array( 'pattern/overrides', $block_id, $attribute_name ), null );
+}
+
+register_block_bindings_source(
+	'core/pattern-overrides',
+	array(
+		'label'              => _x( 'Pattern Attributes', '' ),
```

Comment:
We should probably also update this and pass some context to the `_x( )` function.

Reply:
Gotcha. Updated in f10a4fe1f46f4215141ebf10ddbcdfbaa0f7969b